genie-web/src/main/java/com/netflix/genie/web/configs/GenieApiAutoConfiguration.java

```diff
@@ -73,6 +78,37 @@ public ResourceLoader resourceLoader() {
         return new DefaultResourceLoader();
     }
 
+    /**
+     * Get the protocol resolver that handles files streamed by the agent while a job is running.
+     *
+     * @param agentFileManifestService the file manifest service
+     * @param agentFileStreamService   the file streaming service
+     * @return a {@link AgentFileProtocolResolver}
+     */
+    @Bean
+    @ConditionalOnMissingBean(AgentFileProtocolResolver.class)
+    public AgentFileProtocolResolver agentFileProtocolResolver(
```

Comment:
Looks like we might need a new auto configuration at some point to collect all the `FileServing` related beans